File: Java/Sandbox/test/com/codesignal/AlphabeticShiftTests.java
package com.codesignal;

import org.junit.jupiter.api.Assertions;
import org.junit.jupiter.api.Test;

public class AlphabeticShiftTests {
    @Test
    public void Test1() {
        Assertions.assertEquals("a", Kata.alphabeticShift("z"));
    }

    @Test
    public void Test2(){
        Assertions.assertEquals("dsbaz", Kata.alphabeticShift("crazy"));
    }

    @Test
    public void Test3() {
        Assertions.assertEquals("bbbbcccdde", Kata.alphabeticShift("aaaabbbccd"));
    }

    @Test
    public void Test4() {
        Assertions.assertEquals("gvaaz", Kata.alphabeticShift("fuzzy"));
    }
}

File: SolutionForFun/test/CodeWarsTest/Tests/GeneralKataTests/AlphabeticShiftTests.cs
using CodeWarsTests.Tasks;
using NUnit.Framework;

namespace CodeWarsTests.Tests.GeneralKataTests
{
    [TestFixture]
    public class AlphabeticShiftTests
    {
        [Test]
        public void Test1()
        {
            Assert.AreEqual("a", Kata.AlphabeticShift("z"));
        }

        [Test]
        public void Test2()
        {
            Assert.AreEqual("dsbaz", Kata.AlphabeticShift("crazy"));
        }

        [Test]
        public void Test3()
        {
            Assert.AreEqual("bbbbcccdde", Kata.AlphabeticShift("aaaabbbccd"));
        }

        [Test]
        public void Test4()
        {
            Assert.AreEqual("gvaaz", Kata.AlphabeticShift("fuzzy"));
        }
    }
}
